.travis.yml

```diff
@@ -1,10 +1,6 @@
 language: node_js
 node_js:
-  - 1
```

Comment:
Change the list of tested nodes to the list of [currently supported node versions](https://github.com/nodejs/LTS#release-schedule1).

Reply:
I'm a little confused about how Node handles maintenance for non-LTS releases. Are you sure that 7 is supported and 5 isn't?

Reply:
oh wow, node 7 has already been deprecated!  I thought it was in October for some reason.  Good catch!

Reply:
Yeah I just noticed this section in the LTS:

> An odd-numbered major release will cease to be actively updated when the subsequent even-numbered major release is cut.